### DISCUSSION TITLE
GitHub issue link e.g. #123 is not showing in release notes

### DISCUSSION THREAD

**Discussion by joelberkeley** [community/community#32807] [Projects and Issues] [answered]
In my latest release notes, using #123 to reference issues isn't working. It just doesn't appear on the release notes, as though it's a code comment. It's working for previous releases

**Comment by joelberkeley** [comment, marked as answer]
This now appears to be fixed by itself in my case. Perhaps it was something to do with servers taking time to sync.

**Comment by sonhp19932022** [comment]
Ok
